assets/css/styles.css

```diff
@@ -721,14 +742,275 @@ HAMBURGER MENU DISPLACED DUE TO LINTERS ERROR
   }
 }
 
+/*
+  ADD PROJECT POPUP CSS
+  - Add the Popup class (same as the html file and style it)
+  --  Add .popup button and .popup h1, h2, h3 and p to respective
+      class declarations in the different sections
+  - Add the class open-popup (to be added in the javascript open function)
+*/
+
+.popup-modal {
+  position: absolute;
+  top: 0%;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: var(--image-placeholder-color);
+  z-index: 100;
+}
```

Comment:
Kindly adjust the design of the popup to be as close to the design as possible. The background of the popup is a blur of the page and not given a solid color.

Your design:
![image](https://user-images.githubusercontent.com/20554583/233499043-cac79f7f-9d61-4a1c-9f35-04a64d2ae9f8.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/233499148-f8ef09d2-b3d1-439c-b6ec-a7309681e801.png)

Reply:
@done